### ISSUE TITLE
Error while creating code coverage

### ISSUE THREAD

**Issue by UltiRequiem** [denoland/deno#14397]
As you can see the CI was passing correctly before.

https://github.com/UltiRequiem/eloquent-javascript/commits/main

But since https://github.com/UltiRequiem/eloquent-javascript/commit/a5f766e32333a6a7258d3a6d68717c03064acbc4

While creating code coverage I'm getting this errors:

```sh
deno test --coverage=./cov
```

> This step works as expected

```sh
deno coverage --unstable --lcov ./cov > cov.lcov
```

```
============================================================
Deno has panicked. This is a bug in Deno. Please report this
at https://github.com/denoland/deno/issues/new.
If you can reliably reproduce this panic, include the
reproduction steps and re-run with the RUST_BACKTRACE=1 env
var set and include the backtrace in your report.

Platform: linux x8[6](https://github.com/UltiRequiem/eloquent-javascript/runs/6168539434?check_suite_focus=true#step:8:6)_64
Version: 1.21.0
Args: ["deno", "coverage", "--unstable", "--lcov", "./cov"]

thread 'main' panicked at 'called `Result::unwrap()` on an `Err` value: BadJson(Error("invalid unicode code point", line: 1, column: 2043))', cli/tools/coverage/mod.rs:1[7](https://github.com/UltiRequiem/eloquent-javascript/runs/6168539434?check_suite_focus=true#step:8:7)5:57
note: run with `RUST_BACKTRACE=1` environment variable to display a backtrace
```

But as far I can see on https://github.com/UltiRequiem/eloquent-javascript/commit/a5f766e32333a6a7258d3a6d68717c03064acbc4 there isn't any invalid unicode?

You can see it here too 👇🏽 
https://github.com/UltiRequiem/eloquent-javascript/runs/6168539434?check_suite_focus=true

**Comment by GJZwiers** [comment]
There is a Unicode emoji in `list.ts`: https://github.com/UltiRequiem/eloquent-javascript/blob/a5f766e32333a6a7258d3a6d68717c03064acbc4/src/language/list.ts#L11.

**Comment by bartlomieju** [comment]
Gonna close it as a duplicate of https://github.com/denoland/deno/issues/10936; this will be fixed soon, waiting on `deno_ast` crate upgrade.
